.github/workflows/triage.yml

```diff
@@ -24,11 +24,14 @@ jobs:
           ref: develop # DO NOT CHANGE
 
       - uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065
+      - uses: ./.github/actions/checkout-spack
```

Comment:
It may be good to explicitly configure this to point at the `spack/spack-packages@develop` rather than do the clone inside of the scripts.

Reply:
Right, the part I am talking about is for the packages repo, not spack. I agree we should not be grabbing the head of spack/spack develop.

This line where
```
${{ inputs.path }}/bin/spack repo set --destination "$REPOSITORY_PATH" --scope site builtin
```

For this check we should point at the develop of packages, not the PR, when querying maintainers. This ensures that people can't remove a maintainer without them at least getting a ping.

Then change woiuld be
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
        with:
            ref: develop
            fetch-depth: 1
            path: develop/
      - uses: ./.github/actions/checkout-spack
        with:
            builtin: develop/
```

Reply:
What you're describing already happens in lines 22-24.

Reply:
Oh duh, yea okay then I am fine on this point :)